src/python-fastui/tests/test_forms.py

```diff
@@ -469,3 +469,29 @@ def test_form_textarea_form_fields():
             }
         ],
     }
+
+
+class FormSelectMultiple(BaseModel):
+    values: List[Literal['foo', 'bar']] = Field(title='Select Multiple', description='First Selector')
```

Comment:
surely we need more than one select field in the test to demonstrate that the description doesn't get leaked to subsequent inputs?

Reply:
yes and improved test revealed another source of the problem.. should have been fixed now